packages/dds/sequence/src/test/intervalCollection.fuzz.spec.ts

```diff
@@ -300,7 +312,7 @@ function makeOperationGenerator(
 	const clientBaseOperationGenerator = createWeightedGenerator<Operation, ClientOpState>([
 		[addText, 2, isShorterThanMaxLength],
 		[removeRange, 1, hasNonzeroLength],
-		// [addInterval, 0, all(hasNotTooManyIntervals, hasNonzeroLength)],
+		[obliterateRange, 0, hasNonzeroLength],
```

Comment:
\>0 weight breaks as obliterate doesn't have 0 failures when there are intervals or reconnects

Reply:
i see. in that case, do you think there's value in doing a little refactoring so that the weights are customizable and adding a separate suite for obliterate without those operations?